### DISCUSSION TITLE
Changes are not reflecting in jira

### DISCUSSION THREAD

**Discussion by hima629** [community/community#46902] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when we commit , the commits are not reflecting on Jira? How can I resolve this ?

**Comment by chris-c-thomas** [comment]
Take a look at [Jira's docs](https://support.atlassian.com/jira-cloud-administration/docs/integrate-github-with-jira/) on how to integrate GitHub.

Also note that I think you can only do so with Enterprise accounts.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hima629 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
